File: Flutter/credirama/lib/request/createAccountRequest.dart
import 'dart:core';

class CreateAccountRequest {
    int owner;
    String contract;

  CreateAccountRequest(this.owner, this.contract);

  Map<String, String> toSend() {
    Map<String, String> result = new Map();
    result.putIfAbsent("owner", () => this.owner.toString());
    result.putIfAbsent("contract", () => this.contract);
    return result;
  }

  @override
  String toString() {
    return 'CreateAccountRequest{owner: $owner, contract: $contract}';
  }

}

File: IntegrationTest/src/main/java/fr/unice/polytech/credirama/dto/CreateAccountRequest.java
package fr.unice.polytech.credirama.dto;

import fr.unice.polytech.credirama.dto.contract.Contract;
import lombok.*;

@ToString
@EqualsAndHashCode
@Data
@AllArgsConstructor
@NoArgsConstructor
public class CreateAccountRequest {

    private Contract contract;

    private int owner;

}
